graylog2-web-interface/src/pages/ShowContentPackPage.jsx

```diff
@@ -47,16 +56,18 @@ const ShowContentPackPage = createReactClass({
     if (window.confirm('You are about to delete this content pack, are you sure?')) {
       ContentPacksActions.deleteRev(contentPackId, revision).then(() => {
         UserNotification.success('Content Pack deleted successfully.', 'Success');
-        ContentPacksActions.get(contentPackId);
+        ContentPacksActions.get(contentPackId).catch((error) => {
+          if (error.status !== 404) {
+            UserNotification.error('An internal server error occurred. Please check your logfiles for more information');
+          }
+          history.push(Routes.SYSTEM.CONTENTPACKS.LIST);
+        });
       }, (error) => {
-        /* eslint-disable camelcase */
-        let err_message = error.message;
-        const err_body = error.additional.body;
-        /* eslint-enable camlecase */
-        if (err_body && err_body.message) {
-          err_message = error.additional.body.message;
+        let errMessage = error.message;
+        if (error.responseMessage) {
+          errMessage = error.responseMessage;
         }
-        UserNotification.error(`Deleting bundle failed: ${err_message}`, 'Error');
+        UserNotification.error(`Deleting bundle failed: ${errMessage}`, 'Error');
```

Comment:
"Bundle" was one of the 3 or so terms we used for content packs, so I guess we need to rename that :)

Reply:
:+1: fixed!